app/views/items/show.html.erb

```diff
@@ -21,6 +21,11 @@
 <div class="box">
   <div class="box-header with-border">
     <h3 class="box-title"><%= @item.name %>  <%= item_value(@item.value_in_cents, ', value per Item (in cents): ') %></h3>
+    <small class="box-subtitle">
+      Value per item: <%= @item.value_in_cents ? @item.value_in_cents : 0 %> | 
+      Distribution quantity: <%= @item.distribution_quantity ? @item.distribution_quantity : 0 %> |
+      Package size: <%= @item.package_size ? @item.package_size : 0 %>
```

Comment:
Not a blocker, but for each of these you can just use an `||` fall-through:
```suggestion
      Package size: <%= @item.package_size || 0 %>
```

Reply:
Oh, I didn't know about that! Thanks!!